evm/v0.5/testv2/Schnorr/fixtures/tests.ts

```diff
@@ -1,6 +1,6 @@
 // These tests were generated by by kyber/sign/ethschnorr/ethschnorr_test.go,
 // with printTests set to true.
-const tests = [
+export const tests = [
```

Comment:
Same with this. Seems significantly simpler to just include it in `Schnorr.test.ts`.

Reply:
I wouldn't mind adding them to the same fixture file, i don't think fixtures this large should belong within the same file as the tests though